AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/editor_script_canvas.py

```diff
@@ -37,53 +36,46 @@ class VariableState(Enum):
 class ScriptCanvasComponent:
 
     def __init__(self):
-        self.editor_entity = None
         self.script_canvas_component = None
```

Comment:
Update the initializer to     

```
def __init__(self, editor_entity: EditorEntity) -> None:
        self.component = editor_entity.add_component(SCRIPT_CANVAS_COMPONENT)
```

Reply:
ended up doing this exact thing after acting on the previous comment.